service/security/certificate/api/src/main/java/org/eclipse/kapua/service/certificate/xml/CertificateXmlRegistry.java

```diff
@@ -12,35 +12,40 @@
 package org.eclipse.kapua.service.certificate.xml;
```

Comment:
We are missing the XmlRegistry for the `PublicCertificate`?? 

If yes I would rename this `PrivateCertificateXmlRegistry` and create a new `PublicCertificateXmlRegistry`